src/java/com/puppetlabs/jruby_utils/pool/LockablePool.java

```diff
@@ -122,7 +122,12 @@ E borrowItemWithTimeout(long timeout, TimeUnit unit)
      */
     int currentSize();
 
-   /**
+    /**
+     * Returns the number of available (i.e. unused) Jruby instances in the pool.
+     */
+    int availableJrubies();
```

Comment:
I think we usually capitalize the 'r' in 'JRuby' as well. Let's be consistent.

Reply:
@Magisus I fixed the comment - did you want me to fix the functions too? I wasn't sure.

Reply:
Yes please, that was the one I was most interested in, since it's harder to change later.

Reply:
okay done!